integ-test/src/test/java/org/opensearch/sql/ppl/QueryStringIT.java

```diff
@@ -27,22 +27,29 @@ public void all_fields_test() throws IOException {
 
   @Test
   public void mandatory_params_test() throws IOException {
-    String query = "source=" + TEST_INDEX_BEER + " | where query_string([\\\"Tags\\\" ^ 1.5, Title, 'Body' 4.2], 'taste')";
+    String query =
+        "source="
+            + TEST_INDEX_BEER
+            + " | where query_string([\\\"Tags\\\" ^ 1.5, Title, 'Body' 4.2], 'taste')";
     JSONObject result = executeQuery(query);
     assertEquals(16, result.getInt("total"));
   }
 
   @Test
   public void all_params_test() throws IOException {
-    String query = "source=" + TEST_INDEX_BEER + " | where query_string(['Body', Tags, Title], 'taste beer',"
-        + "allow_leading_wildcard=true, enable_position_increments=true, escape=false,"
-        + "fuzziness= 1, fuzzy_rewrite='constant_score', max_determinized_states = 10000,"
-        + "analyzer='english', analyze_wildcard = false, quote_field_suffix = '.exact',"
-        + "auto_generate_synonyms_phrase_query=true, boost = 0.77,"
-        + "quote_analyzer='standard', phrase_slop=0, rewrite='constant_score', type='best_fields',"
-        + "tie_breaker=0.3, time_zone='Canada/Pacific', default_operator='or',"
-        + "fuzzy_transpositions = false, lenient = true, fuzzy_max_expansions = 25,"
-        + "minimum_should_match = '2<-25% 9<-3', fuzzy_prefix_length = 7)";
+    String query =
+        "source="
+            + TEST_INDEX_BEER
+            + " | where query_string(['Body', Tags, Title], 'taste"
```

Comment:
Can you avoid breaking lines in between of argument and value?

Reply:
I don't believe we can as it will just re-format it on next `spotlessApply`

Reply:
Correction: Yes we can and fixed.